forest/tree/tree.go

```diff
@@ -0,0 +1,58 @@
+package tree
+
+import (
+	cache "github.com/treeverse/lakefs/forest/cache_map"
+	gr "github.com/treeverse/lakefs/graveler"
+	"github.com/treeverse/lakefs/graveler/committed/sstable"
+)
+
+var minimalKey = gr.Key(nil)
+
+type treePartType struct {
+	PartName sstable.ID `json:"part_name"`
+	MaxKey   gr.Key     `json:"max_path"`
+}
+type TreeType struct {
+	treeSlice []treePartType
+}
+
+type treesRepo struct {
+	treesMap   cache.CacheMap
+	partManger sstable.Manager
+}
+
+const (
```

Comment:
What are these? add documentation if they are part of the interface.

Reply:
Will be moved to the cache part